core/src/main/java/io/confluent/kafka/schemaregistry/storage/SchemaRegistry.java

```diff
@@ -33,6 +34,8 @@
 
   SchemaString get(int id) throws SchemaRegistryException;
 
+  boolean hasSubjectConfig(String subject) throws SchemaRegistryStoreException;
```

Comment:
It is ok to not have it in the interface because `ConfigResource` has a schemaRegistry field of type KafkaSchemaRegistry and not just SchemaRegistry, but I wonder if it should have been of type SchemaRegistry instead. Saying it differently, is there any chance of generalizing the SchemaRegistry in ConfigResource? If so, it needs to be part of the interface I think.